ogr/mock_core.py

```diff
@@ -0,0 +1,93 @@
+from typing import Callable, Any, Union
+import functools
+import logging
+from ogr.abstract import PullRequest, PRComment, PRStatus
+import datetime
+
+
+logger = logging.getLogger(__name__)
+
+
+def readonly(*,
+             return_value: Union[Any, None] = None,
+             return_function: Union[Callable, None] = None,
```

Comment:
I think the preferred format is `Optional[Callable]`

Reply:
Okay, sounds reasonable. I've replaced it.